cooperator/__manifest__.py

```diff
@@ -9,16 +9,16 @@
     "summary": "Manage your cooperators",
     "version": "12.0.6.4.2",
     "depends": [
-        "base",
-        "web",
-        "sale",
         "account",
         "base_iban",
-        "product",
+        "mail",
+        "web",
+        # split into cooperator_partner_firstname
```

Comment:
what do these comments mean? i don’t see modules named like this, and if something is split, it should not be there, right? or are these todos?

Reply:
These are todos. Sorry, I did this a bit fast :-)